app/src/main/java/org/schabi/newpipe/local/subscription/SubscriptionFragment.kt

```diff
@@ -207,7 +207,7 @@ class SubscriptionFragment : BaseStateFragment<SubscriptionState>() {
         super.onActivityResult(requestCode, resultCode, data)
         if (data != null && data.data != null && resultCode == Activity.RESULT_OK) {
             if (requestCode == REQUEST_EXPORT_CODE) {
-                val exportFile = Utils.getFileForUri(data.data!!)
+                val exportFile = Utils.getFileForUri(data.data!!)!!
```

Comment:
It seems uneeded:

```
> Task :app:compileDebugKotlin
[...]
w: /home/runner/work/NewPipe/NewPipe/app/src/main/java/org/schabi/newpipe/local/subscription/SubscriptionFragment.kt: (210, 66): Unnecessary non-null assertion (!!) on a non-null receiver of type File
```

Reply:
Oh, I misunderstood the error. Seems like `parentFile` is the one that is nullable, not `exportFile`. _facepalm_